client/components/ReviewRating/ReviewList/Helpfulness.jsx

```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types';
```

Comment:
This is very minor, but can we import React first as a matter of style and consistency?

Reply:
You are right, those are all small things but it is best practice. thanks